src/bundle/Resources/public/js/scripts/core/custom.tooltip.js

```diff
@@ -0,0 +1,28 @@
+(function(global, doc, eZ, $) {
+    eZ.helpers.tooltips = {
```

Comment:
Please follow the convention of other helpers. 
const with method 
using the eZ.addConfig and so